config/fcos/v1_5_exp/translate.go

```diff
@@ -64,6 +64,13 @@ func (c Config) ToIgn3_4Unvalidated(options common.TranslateOptions) (types.Conf
 		return types.Config{}, translate.TranslationSet{}, r
 	}
 	r.Merge(c.processBootDevice(&ret, &ts, options))
+	for _, disk := range ret.Storage.Disks {
+		for _, partition := range disk.Partitions {
+			if partition.Label != nil && *partition.Label == "root" && partition.Number != 4 {
```

Comment:
`root` is the main one, but I think it's reasonable to check for the other reserved partitions as mentioned in https://github.com/coreos/butane/issues/243.  @jlebon, WDYT?

Reply:
Yup, that makes sense to me!